package.json

```diff
@@ -32,9 +32,9 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/hpcc-systems/Visualization"
+    "url": "git+https://github.com/hpcc-systems/Visualization.git"
```

Comment:
I'm not sure what this "git+https" syntax is... but I assume that was intentional.